src/main/java/it/rate/webapp/services/GoogleImageService.java

```diff
@@ -52,6 +53,18 @@ public String savePlaceImage(MultipartFile image, Long placeId) throws IOExcepti
     }
   }
 
+  public byte[] getImageById(String imageId) throws IOException {
+
+    try {

```

Comment:
Does try inside of another try do something special?

Reply:
whole method changed to implement new exceptions